### ISSUE TITLE
Build is broken

### ISSUE THREAD

**Issue by bugged84** [aimacode/aima-csharp#7]
It appears there are java language constructs (e.g. List<? extends Sentence>) sprinkled throughout the code base, as well as missing assemblies and class definitions for referenced types. How is anyone else building this solution? Seems like it's been this way all year if not longer.

**Comment by ShobhitLamba** [comment]
Hello bugged84,
Yes I am aware that the build is broken right now. I never got around to fixing it since I got busy with other pressing academic commitments. You are more than welcome to fix things and contribute to the project.
As for code snippets resembling java language syntax, it might be possible that I missed out on rectifying those while I was working on it. Since the C# repo inherits much from the java repository, I used the existing java code to port it to C#. So, there might be issues like you mentioned that I might have accidentally missed out on.
